grid/chat/channels/maps/slack.suma

```diff
@@ -0,0 +1,212 @@
+profile = "chat/channels@1.0"
+provider = "slack"
+
+"""
+GetChannels map
+"""
+map GetChannels {
+  types = call typesToQuery(types = input.types)
+
+  http GET "/conversations.list" {
+    request "application/x-www-form-urlencoded" {
+      query {
+        cursor = input.page
+        exclude_archived = true
+        limit = input.limit
+        types = types
+      }
+
+      headers {
+        "Authorization" = `Bearer ${parameters.accessToken}`
+      }
+    }
+
+    response 200 "application/json" {   
+      error = call MapSlackError(
+        statusCode = statusCode,
+        error = body,
+        headers = headers,
+      ) if (!body.ok)
+
+      return map error if (!body.ok) error
+
+      channels = call MapChannels(channels = body.channels)
+      rateLimit = call MapRateLimitDetails(headers = headers)
+      result.channels = channels
```

Comment:
Why not assign result.channels directly? Also am I mad or I don't see any `result` constructor? Is that some implicit Comlink feature? :grimacing:

Reply:
Yeah it's a feature 😄 This is needed because of resolving objects is not really finished in linter, when one object switches from jessie expression to comlink object and vice versa, it doesn't keep properties from first type of assignment.